app/styles/main.scss

```diff
@@ -1,105 +1,163 @@
-  color: $white;
+color: $white;
 }
 
 .btn-head {
-  font-size: 100%;
+font-size: 100%;
 }
 
 .carousel {
-  margin-top: 20px;
+margin-top: 20px;
 }
 
 h2 {
-  margin: 0;
-  color: #666;
-  padding-top: 90px;
-  font-size: 52px;
-  font-family: "trebuchet ms", sans-serif;
+margin: 0;
+color: #666;
+padding-top: 90px;
+font-size: 52px;
+font-family: "trebuchet ms", sans-serif;
 }
 
 .item {
-  background: #333;
-  text-align: center;
-//  height: 300px !important;
+background: #333;
+text-align: center;
+height: 300px !important;
 }
 
 .pad-head {
-  padding: 30px 10px 0px 10px;
+padding: 30px 10px 0px 10px;
 }
 
 .row-head {
-  background-color: $white;
-  height: 200px;
+background-color: $white;
+height: 200px;
 }
 
 .main_title {
-  // background-color: white;
-  // color: green;
-  -webkit-box-shadow: inset 0 0 100px rgba(0,0,0,.5);
-
-  box-shadow: inset 0 0 100px rgba(0,0,0,.5);
-
-  h1{
-    // color: black;
-  }
-
-  .btn-danger {
-    width: auto;
-    height: auto;
-  }
+background-color: white;
+color: black;
+-webkit-box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 
-  text-align: center;
+box-shadow: inset 0 0 100px rgba(0,0,0,.5);
 
+h1{
+color: black;
 }
 
-// .navbar-default navbar-collapse in {
-//   border-color: blue;
-// }
-.landing-title{
-  color: $grey;
+.btn-danger {
+width:auto;
+height: auto;
 }
 
-#main_content {
-  padding-right: 18px;
-  padding-left: 18px;
-}
+text-align: center;
 
-
-#left1{
-  float: left;
-  width: 100%;
 }
+#main_content {
+margin-right: 18px;
+margin-left: 18px;
+}
+// need to re-name
+.col-sm-4 {
+width: 33.33333333%;
+color: black;
+}
+.navbar-default {
+background-color: black;
+border-color: black;
+height: 100px;
+}
+.col-sm-8{
+text-align: left;
+}
+.col-sm-3{
+margin-top: 18px;
+width: 30%;
+}
+
+.navbar .navbar-form{
+padding: 0 15px;
+border: 0;
+-webkit-box-shadow: none;
+box-shadow: none;
+}
+
+#aboutustxt{
+width:900px;
+text-align:justify;
+}
+
+#navbar-spacing{
+margin-left: 5px;
+margin-right: 5px;
+margin-top: 18px;
+margin-bottom: 18px;
+}
+
+#form-spacing{
+  width: 50%;
+  display: block;
+  height: 34px;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #555;
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  -webkit-transition: border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;
+  -o-transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+  transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
+}
+
+#form-spacing2{
+  display: block;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #555;
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
```

Comment:
`.075` should be written with a leading zero as `0.075`<br>Commas in function arguments should be followed by a single space